config/wiki.yaml

```
@@ -46,7 +46,7 @@ affinity:
     requiredDuringSchedulingIgnoredDuringExecution:
       - labelSelector:
           matchExpressions:
-            - key: app
+            - key: name
```

```suggestion
            - key: "app.kubernetes.io/name"
```

I believe we should use the full key name. Currently, the `metadata.labels` of each pod contains:

```
app.kubernetes.io/instance: wiki
app.kubernetes.io/name: wiki
```